gallery.html

```diff
@@ -1,100 +1,98 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <link rel="icon" type="image/x-icon" href="favicon.ico" />
-    <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
-    <link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png" />
-    <link rel="manifest" href="site.webmanifest" />
-    <link rel="mask-icon" href="safari-pinned-tab.svg" color="#5bbad5" />
-    <meta name="msapplication-TileColor" content="#da532c" />
-    <meta name="theme-color" content="#ffffff" />
-    <meta name="author" content="Matthew Burgan" />
-    <meta name="description" content="this is an awesome website!" />
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles/normalize.css">
-    <link rel="stylesheet" href="styles/main.css">
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-   <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@100&display=swap" rel="stylesheet">
-   <link rel="preconnect" href="https://fonts.gstatic.com">
-<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
-    <title>CSD120 - Matthew Burgan</title>
-  </head>
-  <body>
-    <div class="width-wrap">
-      <header id="top">
-        <nav>
-          <ul>
-            <li><a href="index.html">Home</a></li>
-            <li><a href="links.html">Links</a></li>
-            <li><a href="glossary.html">Glossary</a></li>
-          </ul>
-        </nav>
-      </header>
-<h1> Gallery </h1>
-<div class="Gallery">
-<a
-class="row-span2">
-<a href="https://loremflickr.com/1024/1024/?lock=40"> 
-<img src="https://loremflickr.com/300/600?lock=40" />
-<a href="https://loremflickr.com/1024/1024/?lock=41"> 
-<img src="https://loremflickr.com/300/300?lock=41" />
-<a href="https://loremflickr.com/1024/1024/?lock=42"> 
-<img src="https://loremflickr.com/300/300?lock=42" />
-<a href="https://loremflickr.com/1024/1024/?lock=43"> 
-<img src="https://loremflickr.com/300/300?lock=44" />
-<a href="https://loremflickr.com/1024/1024/?lock=44"> 
-<img src="https://loremflickr.com/300/300?lock=45" />
-<a href="https://loremflickr.com/1024/1024/?lock=45"> 
-<img src="https://loremflickr.com/300/300?lock=46" />
-<a href="https://loremflickr.com/1024/1024/?lock=46"> 
-<img src="https://loremflickr.com/300/300?lock=47" />
- <a href="https://loremflickr.com/1024/1024/?lock=47"> 
- <img src="https://loremflickr.com/300/300?lock=48" />
-<a href="https://loremflickr.com/1024/1024/?lock=48"> 
```

Comment:
Lots of formatting and validation problems here!  You didn't close any of your <a> tags